install/helm/open-match/templates/podsecuritypolicy.yaml

```diff
@@ -14,10 +14,45 @@
 
 {{- if index .Values "open-match-core" "enabled" }}
 {{- if empty .Values.ci }}
+# om-redis-podsecuritypolicy is the least restricted PSP used to create privileged pods to disable THP in host kernel.
+apiVersion: policy/v1beta1
```

Comment:
Doesn't Redis already come with a PSP?

Reply:
It comes with a service-account along with some configured rolebindings but doesn't have a PSP by default https://github.com/helm/charts/tree/master/stable/redis/templates